Add link to a set of Spring-based AWS Lambda samples (#425)
awsdocs/aws-lambda-developer-guide: doc_source/lambda-samples.md

**Sample Lambda applications in Java**
+ [java\-basic](https://github.com/awsdocs/aws-lambda-developer-guide/tree/main/sample-apps/java-basic) – A collection of minimal Java functions with unit tests and variable logging configuration\.

> + [spring\-samples](https://github.com/spring-cloud/spring-cloud-function/tree/3.2.x/spring-cloud-function-samples/function-sample-aws) – A set of example demonstrating several ways to benefit from using [Spring Cloud Function](https://spring.io/projects/spring-cloud-function) framework when working with AWS Lambdas\.

Reviewer: ```suggestion
+ [spring\-samples](https://github.com/spring-cloud/spring-cloud-function/tree/3.2.x/spring-cloud-function-samples/function-sample-aws) – An example demonstrating how to use [Spring Cloud Function](https://spring.io/projects/spring-cloud-function) framework when working with AWS Lambda functions\.
```
